CONTRIBUTING.md

```diff
@@ -0,0 +1,126 @@
+# Intro
+
+If there is a bugfix or feature you'd like to see implemented, the best place to start is with a GitHub issue. First check there is not an issue raised already. If the issue has not yet been raised, it is worth raising one before making any changes yourself. That way, we can let you know whether or not this is something we are already working on.
+
+As an opensource project, 51Degrees accepts changes from the community. To propose a change:
+1. Commit your change locally to a branch based off `main`,
+2. Push to GitHub,
+3. Open a pull request to the `main` branch in the 51Degrees GitHub,
+4. 51Degrees will assess and update.
+
+The rest of this document goes into more details of what is required.
+
+The reader should be familiar with:
+- [Git](https://git-scm.com/doc)
+- [GitHub Issues](https://github.com/features/issues)
+- [GitHub Pull Requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)
+
+# Contents
+- [Issues](#issues)
+- [Changes](#changes)
+  - [Branching Strategy](#branching-strategy)
+  - [Commenting](#commenting)
+  - [Code Style](#code-style)
+  - [Tests](#tests)
+  - [Commit Messages](#commit-messages)
+- [Making a Pull Request](#making-a-pull-request)
+  - [Merging Policy](#merging-policy)
+  - [Pull Request Message](#pull-request-message)
+- [Build Process](#build-process)
+
+# Issues
+
+Issues should be raised through the GitHub issues system. These are then reviewed weekly by 51Degrees.
+
+An issue should contain a complete explanation of the problem, including the environment being used.
+It should also show the steps needed to recreate the issue.
+
+# Changes
+
+## Branching Strategy
+
+In general, branches should be taken from `main`, and PRs created back to `main` for each feature. For a more in depth descriptions, see [Feature Branch Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow").
```

Comment:
I think this needs to tell people what their branches should be called.
You might not want the top level flooded with branches from people. If that's the case then we definitely want to let people know what directory to put their branches in.
In any case, naming conventions should to be defined here (assuming we actually care about branch naming?).

Maybe something like:
[contributor-name]/[change-name]

lower case, kebab case is preferred.